src/test/scala/net/snowflake/spark/snowflake/MiscSuite01.scala

```diff
@@ -208,6 +209,9 @@ class MiscSuite01 extends FunSuite with Matchers {
     assert(metric.get(TelemetryClientInfoFields.SPARK_CONNECTOR_VERSION).asText().equals(Utils.VERSION))
     // check one JVM option
     assert(metric.get(TelemetryClientInfoFields.MAX_MEMORY_IN_MB).asLong() > 0)
+    assert(metric.get(TelemetryClientInfoFields.SPARK_APPLICATION_ID).asText().equals(
+      SparkEnv.get.conf.get("spark.app.id")))
+    assert(!metric.get(TelemetryClientInfoFields.IS_PYSPARK).asBoolean())
```

Comment:
To reviewer:
It is difficult to add unit test to cover `is_pyspark` to be true. 
but I manually tested that `is_pyspark` is set as true if pyspark is used.

Reply:
please manually run some tests in paspark, and verify the telemetry from snowhouse, to see if everything is correct.

Reply:
Below is a sample with pyspark. It looks good.
```
{
  "data": {
    "application_name": "SnowflakeSourceSuite",
    "certified_jdbc_version": "3.13.9",
    "cpu_cores": 16,
    "free_memory_in_mb": 1187,
    "is_pyspark": false,
    "java_version": "11.0.12",
    "jdbc_version": "3.13.9",
    "max_memory_in_mb": 16384,
    "os_name": "Mac OS X",
    "scala_version": "2.12.11",
    "sfurl": "sfctest0.us-central1.gcp.snowflakecomputing.com",
    "spark_application_id": "local-1635800158078",
    "spark_config": {
      "spark.app.id": "local-1635800158078",
      "spark.app.name": "SnowflakeSourceSuite",
      "spark.driver.host": "ip-192-168-0-117.us-west-2.compute.internal",
      "spark.executor.id": "driver",
      "spark.master": "local"
    },
    "spark_connector_version": "2.9.2",
    "spark_version": "3.1.1",
    "total_memory_in_mb": 1232
  },
  "source": "spark_connector",
  "type": "spark_client_info"
}
```